test/licensemaster/lm_c3_test.go

```diff
@@ -87,4 +93,105 @@ var _ = Describe("Licensemaster test", func() {
 			testenv.VerifyLMConfiguredOnPod(deployment, searchHeadPodName)
 		})
 	})
+
+	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
+		It("licensemaster: Splunk Operator can configure a C3 SVA and have apps installed locally on LM", func() {
+
+			var (
+				appListV1        []string
+				testS3Bucket     = os.Getenv("TEST_INDEXES_S3_BUCKET")
+				testDataS3Bucket = os.Getenv("TEST_BUCKET")
+				s3AppDirV1       = "appframework/regressionappsv1/"
+				currDir, _       = os.Getwd()
+				downloadDirV1    = filepath.Join(currDir, "lmV1-"+testenv.RandomDNSName(4))
+			)
+
+			// Parse Applist1 to create Valid Apps1
```

Comment:
Nit: Dont need a comment ... However the value is being assigned so comment is redundant.